lte/gateway/c/core/oai/test/amf/test_amf_procedures.cpp

```diff
@@ -196,6 +202,73 @@ bool validate_smc_procedure(
   return true;
 }
 
+TEST_F(AMFAppProcedureTest, TestRegistrationAuthSecurityModeReject) {
+  amf_ue_ngap_id_t ue_id = 0;
+  std::vector<MessagesIds> expected_Ids{
```

Comment:
where is this used?

Reply:
This UT is used when authentication response comes with a security mode reject(case 24).

Reply:
I have removed unused code.